src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -1,8 +1,4 @@
-export const description = `
-createBindGroup validation tests.
-
-TODO: review existing tests, write descriptions, and make sure tests are complete.
```

Comment:
Just to double check before we remove this, do you think this area is completely tested aside from outstanding TODOs?

Reply:
I.e. do we test every validation rule in createBindGroup() - I haven't reviewed the spec, so not sure. (Sorry I forgot about this until now)

Reply:
Added the todo back indicating I still need to check to ensure all validation rules are covered.